app/src/models/popup.ts

```diff
@@ -121,3 +124,12 @@ export type Popup =
       pullRequest: PullRequest
     }
   | { type: PopupType.MergeConflicts; repository: Repository }
+  | {
+      type: PopupType.OversizedFiles
+      dispatcher: Dispatcher
+      fileList: string[]
+      commitSummary: string
+      commitDescription: string | null
+      repository: Repository
+      trailers?: ReadonlyArray<ITrailer>
```

Comment:
Seeing these three fields `commitSummary`, `commitDescription` and `trailers` repeated here makes me think this is a great candidate for turning them into a shape like `CommitDetails` - for now I won't hold up this PR but I'll follow up with some tidy-up to introduce this shape:

```ts
export type CommitDetails = {
  summary: string
  description: string | null
  trailers?: ReadonlyArray<ITrailer>
}
```